app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -8,10 +15,10 @@ export default function NoteTable({
 }) {
   return (
     <div className="w-full px-4">
-      <table className="nui-table nui-table--checkbox">
+      <table className="nui-table nui-table--checkbox table-fixed">
```

Comment:
Please add `nui-table--hover nui-table--actions` classname for hover effects and for the `IconButtons` to be visible only during hover.

Reply:
Resolved in #9